packages/utils/adapters/package.json

```diff
@@ -26,7 +26,8 @@
     "@uifabricshared/eslint-config-rules": "^0.1.1",
     "react-native": "0.62.2",
     "react-native-web": "^0.12.2",
-    "react-native-windows": "0.0.0-master.47"
+    "react-native-windows": "0.0.0-master.47",
+    "react-dom": "^16.13.1"
```

Comment:
So as far as this hack goes, have you tried adding the react-dom dep as a devdep in the root package.json. I'm not sure whether that's the recommended fix or not, but is likely cleaner than having it here if it in fact works.